privilege/privileges/privileges.go

```diff
@@ -139,6 +140,7 @@ func (p *UserPrivileges) ConnectionVerification(user, host string, authenticatio
 	if len(pwd) == 0 && len(authentication) == 0 {
 		p.user = user
 		p.host = host
+		p.authHost = h
```

Comment:
Should we directly set `p.user = u` and `p.host = h`?
Where is `p.host` used after this change?

Reply:
fixed, PTAL @tiancaiamao